charts/telemetry-server/templates/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: tls-telemetry-ingress
+spec:
+  secretName: tls-telemetry-ingress
+  duration: 720h # 30 days. The max duration is set by the CA at 30 days.
+  renewBefore: 168h # 7d
+  subject:
+    organizations:
+      - telemetry.op-aws-eu-dmz-1-ingress.op.suse.org
```

Comment:
This setting is deployment specific, and probably should be specified by the deployment env settings, e.g. override-values.yaml, rather than hard coded.